components/infobox/wikis/criticalops/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,130 @@
+---
+-- @Liquipedia
+-- wiki=criticalops
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Page = require('Module:Page')
+local TeamHistoryAuto = require('Module:TeamHistoryAuto')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector')
+local Player = Lua.import('Module:Infobox/Person')
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local ROLES = {
+	-- Players
+	['igl'] = {category = 'In-game leaders', variable = 'In-game leader'},
+
+	-- Staff and Talents
+	['analyst'] = {category = 'Analysts', variable = 'Analyst', staff = true},
+	['broadcast analyst'] = {category = 'Broadcast Analysts', variable = 'Broadcast Analyst', talent = true},
+	['observer'] = {category = 'Observers', variable = 'Observer', talent = true},
+	['host'] = {category = 'Host', variable = 'Host', talent = true},
+	['coach'] = {category = 'Coaches', variable = 'Coach', staff = true},
+	['caster'] = {category = 'Casters', variable = 'Caster', talent = true},
+}
+
+---@class COPSInfoboxPlayer: Person
+---@field role {category: string, variable: string, staff: boolean?, talent: boolean?}?
+---@field role2 {category: string, variable: string, staff: boolean?, talent: boolean?}?
+local CustomPlayer = Class.new(Player)
+local CustomInjector = Class.new(Injector)
+
+---@param frame Frame
+---@return Html
+function CustomPlayer.run(frame)
+	local player = CustomPlayer(frame)
+	player:setWidgetInjector(CustomInjector(player))
+
+	player.args.history = TeamHistoryAuto.results{convertrole = true}
+	player.role = player:_getRoleData(player.args.role)
+	player.role2 = player:_getRoleData(player.args.role2)
+
+	return player:createInfobox()
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	local caller = self.caller
+	local args = caller.args
+
+	if id == 'status' then
+		return {
+			Cell{name = 'Status', content = caller:_getStatusContents()},
```

Comment:
With the current setup on the wiki, it's no different from using commons for status, so pelase do that